assets/css/styles.css

```diff
@@ -501,3 +501,18 @@ button.bg-primary:focus {
 ::-webkit-scrollbar-thumb:hover {
   background: #555; 
 }
+
+img {
+  user-drag: none !important;
+  user-select: none !important;
+  -moz-user-select: none !important;
+  -webkit-user-drag: none !important;
+  -webkit-user-select: none !important;
+  -ms-user-select: none !important;
+  -webkit-touch-callout: none !important;
+  -webkit-user-select: none !important;
+  -khtml-user-select: none !important;
+  -moz-user-select: none !important;
+  -ms-user-select: none !important;
+   user-select: none !important;
```

Comment:
these seem to be duplicate entries of the same thing.

Reply:
these are required for the different browsers

Reply:
```
$ sed -n 510p assets/css/styles.css |shasum 
5f2791809bee9b05ab16c9e243e07bb0ccbb3ddb  -
$ sed -n 513p assets/css/styles.css |shasum 
5f2791809bee9b05ab16c9e243e07bb0ccbb3ddb  -
```

the lines are exactly the same?

Reply:
> these are required for the different browsers

Yes, understood, but lines 510 and 513 are both "-webkit-user-select: none !important;"

Reply:
Oh, I didn't saw that. let me fix it

Reply:
the same for:
 
- 508 and 514
- 511 and 515

the rest is fine.

Reply:
Fixed that one too. Sorry for the trouble, I guess I pasted it 2 times.